gst/nnstreamer/tensor_transform/tensor_transform.c

```diff
@@ -200,7 +200,7 @@ gst_tensor_transform_mode_get_type (void)
             "option=D1\':D2\':D3\':D4 (fixed to 3)",
           "transpose"},
       {GTT_STAND, "Mode for statistical standardization of tensor, "
-            "option=default",
+            "option=default[:TYPE]",
```

Comment:
This might be a bit difficult for users to understand.
Please update ```gst/nnstreamer/tensor_transform/README.md``` as well.

Reply:
I will update this on another PR (#3129 to add new option 'DC remove')